src/DynamoCoreWpf/ViewModels/Search/BrowserItemViewModel.cs

```diff
@@ -512,6 +512,13 @@ public ClassInformationViewModel ClassDetails
 
     public class ClassesNodeCategoryViewModel : NodeCategoryViewModel
     {
-        public ClassesNodeCategoryViewModel() : base("") { }
+        public NodeCategoryViewModel Parent { get; private set; }
+
+        public ClassesNodeCategoryViewModel(string fullCategoryName, NodeCategoryViewModel parent)
```

Comment:
Do we need to pass `fullCategoryName` here? It seems to always be `Configurations.ClassesDefaultName` for this class.